src/Hydra/JsonSchema/SchemaFactory.php

```diff
@@ -26,6 +27,9 @@
  */
 final class SchemaFactory implements SchemaFactoryInterface, SchemaFactoryAwareInterface
 {
+    use HydraPrefixTrait;
+    public const HYDRA_PREFIX = 'hydra:';
+    public const HYDRA_CONTEXT_HAS_PREFIX = 'hydra_prefix';
```

Comment:
Couldn't we have these constants at only one place for all the component instead of duplicating them in all classes?

Reply:
PHP 8.1 doesn't support constants inside traits...